Makefile

```diff
@@ -20,7 +20,7 @@ all:
 	@echo or set BUILDTAG to a specific tag
 
 # Build all binaries
-build: build_linux build_win
+build: build_linux build_win build_mac_intel build_mac_apple
```

Comment:
I wonder if using `build_mac_amd64` and `build_mac_arm64`  would be slightly better because it refers to the GOARCH values. What do you think?

Reply:
Technically, they are better. But in a quick look, `arm64` and `amd64` are easily mixed up.
But if you think, they are better, that's OK for me.

Reply:
As we use "amd64" for the distribution packages in a release, I think it is slightly better to use them here as well. But as always I think a lot about the right consistent naming... ;)